schema.sql

```diff
@@ -63,7 +64,12 @@ create table fileruns (
 drop table if exists access;
 create table access (
   access_id integer primary key autoincrement,
-  site_access_code varchar not null,
+  server_id varchar not null,
+  server_name varchar,
+  server_address varchar,
+  user_id varchar not null,
+  user_name varchar,
+  user_email varchar,
```

Comment:
Actually, let's chat tomorrow about whether or not we want to support proper authentication with something like Flask-Auth.